apps/src/util/HttpClient.ts

```diff
@@ -74,12 +74,13 @@ async function fetchJson<ResponseType>(
 }
 
 /**
- * POST to the given endpoint. Adds the Rails authenticity
+ * o to the given endpoint. Adds the Rails authenticity
```

Comment:
nit: update comment - maybe "Sends a request to the given endpoint"?

Reply:
Oh yeah! Forgot to fix this - updated. Thanks!

Reply:
And thanks for pairing with me on this @sanchitmalhotra126 !